README.md

```diff
@@ -11,6 +11,12 @@ dotnet restore
 dotnet tool restore
 ```
 
+- Formatter install/documentation
+```
+dotnet tool install -g dotnet-format
+```
+https://github.com/dotnet/format
```

Comment:
I have added a new section `Tech Docs` in this README file, please merge latest commits in main branch and add this link in the new section

At the same time, you can fix the conflicts currently reported in this file